src/scss/components/_aside.scss

```diff
@@ -10,39 +10,51 @@ aside {
 
 		nav {
 			ul {
-				border-left: 1px solid $green-dark;
+				box-shadow: rem(1) 0 0 0 inset $green-dark;
 				list-style-type: none;
 				margin-left: rem(45);
 				padding: 0 rem(0);
+				position: -webkit-sticky;
+				position: sticky;
+				top: rem(58);
 				width: rem(195);
 
 				li {
-					line-height: rem(34);
+					left: rem(-2);
 					margin: 0 0 rem(23) 0;
-					padding-left: rem(30);
+					padding-left: rem(23);
 					width: rem(151);
 					word-wrap: break-word;
 
 					&:active {
-						border-left: rem(5) solid $green-dark;
-						font-weight: $font-weight-bold;
-						left: rem(-3);
-						margin: 0 0 rem(23) 0;
-						padding-left: rem(30);
+						box-shadow: rem(5) 0 0 0 inset $green-dark;
+						padding-left: rem(25);
 						position: relative;
 					}
 
 					a {
 						color: $navy-light;
 						font-size: rem(24) !important; // Enactors.css in infusion uses !important on this element
 						font-weight: $font-weight-normal;
+					}
 
-						&:active {
+					&:first-child {
+						a {
 							font-weight: $font-weight-bold;
```

Comment:
As now the design is ready, h1 should have bigger font size instead of the font weight in bold.

Reply:
Which designs are you referring to? In [Figma](https://www.figma.com/file/0lcLol3X5MmOXackT2YbHJ/WeCount-website?node-id=1622%3A5647) this is what is laid out. The only difference is that I kept all sticky menu items bold in enhance inputs. @uttaraghodke thoughts?